ghost/core/core/server/data/schema/schema.js

```diff
@@ -422,6 +422,7 @@ module.exports = {
             defaultTo: 'none',
             validations: {isIn: [['public', 'none']]}
         },
+        trial_period_days: {type: 'integer', unsigned: true, nullable: false, defaultTo: 0},
```

Comment:
The naming `trial_period_days` matches naming used by Stripe for passing trial days value for subscriptions/sessions.

Reply:
@allouis qq: what do you think about matching Stripe here? I am on the fence.

Reply:
Given that this table is for storing Tiers which are a Ghost concept, there's no reason to match Stripe naming IMO.

However, I don't think the name is necessarily bad, but if there is a better name we should go for it!

All I can think is `trial_period` or `trial_days` which is a lil shorter 🦐

Reply:
@ErisDS Do you have any pref on the naming between these options ? :)

Reply:
`trial_days` feels nicest to me

Reply:
👌 Yeah sounds good to me too, will update, thanks.